src/main/java/com/flansmod/common/guns/EntityBullet.java

```diff
@@ -1491,4 +1533,210 @@ public void readSpawnData(ByteBuf data) {
     public boolean isBurning() {
         return false;
     }
+    
```

Comment:
I'd be in favour to moving this to a static class in the utility section of the mod, but that can be done later.